flows/flows.py

```diff
@@ -501,10 +531,57 @@ def update_data_summary(self, dataframes_dict: dict) -> dict:
 
         function_id = "8"
 
-        self.columns_set = detect_columns_types_summary(dataframes_dict, threshold=50)
+        self.columns_set = detect_columns_types_summary(dataframes_dict, threshold=self.categorical_threshold)
         try:
             self.guidance(self.flow_steps[function_id])
         except Exception as e:
             print(f"It seems that the next step is not defined. Error{e}")
 
         return self.columns_set
+
+    def save_post_processed_data(self, dataframe_dict: dict,
+                                 path: str,
+                                 suffix: str = "",
+                                 directory_name: str = "post_processing"):
+        """ Data saver
+
+        The function saves data after applying the processing functions.
+
+        :param dict dataframe_dict: A dictionary that contains Pandas dataframes  e.g. dataframes_dict={"train":
+                train_dataframe, "test": test_dataframe}
+        :param str path: The path to the data.
+        :param str suffix: A string that can be added to the filename to get unique names.
+        :param str directory_name: The subdirectory where the post-processing data will be saved
+        :return:
+        """
+
+        function_id = "9"
+
+        path = os.path.join(path, directory_name)
+        print(f"Check if the directory {path} exists")
+
+        if not os.path.exists(path):
+            os.makedirs(path)
```

Comment:
you could also try the following one-liner: 
`os.makedirs(path, exist_ok=True)`